scripts/build_electron_app.py

```diff
@@ -55,38 +57,58 @@ def get_ignore_regex():
 
     ignore_flags = []
     for regex in ignore_list:
-        ignore_flags.extend(["--ignore", regex])
+        ignore_flags.extend(["--ignore="+regex])
 
     return ignore_flags
 
 def get_platform():
-    os_type = platform.system()
-    if os_type  == 'Windows':
+    os_type = platform.system().lower()
+    if os_type  == 'windows':
         return 'win32'
-    elif os_type == 'Darwin':
+    elif os_type == 'darwin':
         return 'darwin'
     else:
         raise SystemExit(script_tab + 'Unsupported OS {}'.format(os_type))
 
 def build_electron_app():
     print(script_tag + "Running electron-packager process.")
 
-    process_args = [
-        shutil.which("electron-packager"),
-        project_root_dir,
-        "OpenTrons",
-        "--platform", get_platform(),
-        "--arch", get_arch(),
-        "--out", output_dir,
-        "--icon", get_icon_path(),
-        "--asar", "true",
-        "--overwrite",
-        "--prune",
-    ] + get_ignore_regex()
-
-
-    electron_packager_process = subprocess.Popen(process_args)
-    electron_packager_process.communicate()
+    os_type = get_platform()
+    print(script_tag + "os_type: {}".format(os_type))
+    if os_type == "darwin":
+        process_args = [
+            shutil.which("electron-packager"),
+            project_root_dir,
+            "OpenTrons",
+            "--platform", get_platform(),
+            "--arch", get_arch(),
+            # SETTING VERSION IS CURRENTLY FAILING ON TRAVISß
```

Comment:
can you provide more info how this is failing on Travis? The current master branch of travis has this set and it's not failing (unless i'm wrong and i could be)..